core/src/fgm/fgj/gamejamgame/GameEvent.java

```diff
@@ -364,136 +405,89 @@ private void handleCrewMemberExchange(Ship ship) {
-			}
-
-			else if (eventKey == 5) {
-				List<CrewMember> crewMembers = ship.listCrewMembers();
-				int crewMemberIndex = rand.nextInt(crewMembers.size());
-				int foodEaten = rand.nextInt(3) + 1;
-				this.eventText = "Crew member '" + crewMembers.get(crewMemberIndex).getName() + "' returned from planet '" + planet.getName() + " famished and ate " + foodEaten + " extra food from the cargo bay.";
-				this.isPositive = true;
-			}
-
-			else if (eventKey == 6) {
-				handleAmmoProductEvent(ship);
-			}
-
-			else {
-				this.eventText = "Nothing happened";
-				this.isPositive = true;
-			}
-		} else if (eventContext == EventContext.SOLAR_SYSTEM) {
-			if (eventKey == 0) {
-				this.eventText = "There is a solar storm. You must leave this system temporarily.";
-				this.isPositive = false;
-			}
-
-			else if (eventKey == 1) {
-				int fuelAmount = rand.nextInt(3) + 1;
-				this.eventText = "You found an abandoned coal bunker floating in space. You were able to salvage " + fuelAmount + " units of fuel from it.";
-				ship.getCargoBay().increaseFuel(fuelAmount);
-				this.isPositive = true;
-			}
+	private void handleEarthquakeEvent() {
+		int damageAmount = new Random().nextInt(10) + 1;
+		Boolean isShipDestroyed = this.ship.issueHullDamage(damageAmount);
+		this.isPositive = false;
+		if (isShipDestroyed) {
+			this.eventText = "An earthquake occurred. Your ship took" + damageAmount + " damage and was destroyed.";
+			this.didLose = Boolean.TRUE;
+		} else {
+			this.eventText = "An earthquake occurred. Your ship took" + damageAmount + " damage.";
+		}
+	}
 
-			else if (eventKey == 2) {
-				int foodAmount = rand.nextInt(3) + 1;
-				this.eventText = "Rats ate " + foodAmount + " units of your food.";
-				ship.getCargoBay().decreaseFood(foodAmount);
-				this.isPositive = false;
-			}
+	private void handleExtraEatingEvent() {
+		Random rand = new Random();
+		List<CrewMember> crewMembers = this.ship.listCrewMembers();
+		int crewMemberIndex = rand.nextInt(crewMembers.size());
+		int foodEaten = rand.nextInt(3) + 1;
+		this.eventText = "Crew member '" + crewMembers.get(crewMemberIndex).getName() + "' returned from planet '" + this.planet.getName() + "' famished and ate " + foodEaten + " extra food from the cargo bay.";
+		this.ship.getCargoBay().decreaseFood(foodEaten);
+		this.isPositive = false;
+	}
 
-			else if (eventKey == 3) {
-				this.eventText = "You found an abandoned ship adrift in space and ransacked its cargo bay!";
-				ship.getCargoBay().increaseMetal((int)(Math.random() * 6));
-				ship.getCargoBay().increaseWater((int)(Math.random() * 6));
-				ship.getCargoBay().increaseAmmo((int)(Math.random() * 6));
-				ship.getCargoBay().increaseFood((int)(Math.random() * 6));
-				ship.getCargoBay().increaseFuel((int)(Math.random() * 6));
-				this.isPositive = true;
-			}
+	private void handleSolarStormEvent() {
+		this.eventText = "There is a solar storm. You must leave this system temporarily.";
+		this.isPositive = false;
+	}
```

Comment:
I feel like these are very similar to the radiation events, but are less interesting.